x-pack/platform/plugins/shared/osquery/server/routes/action_results/get_action_results_route.ts

```diff
@@ -122,27 +175,17 @@ export const getActionResultsRoute = (
           const aggsBuckets =
             res.rawResponse?.aggregations?.aggs.responses_by_action_id?.responses.buckets;
 
-          const previousEdges =
-            agentIds?.map(
-              (agentId) =>
-                ({ fields: { agent_id: [agentId] } } as unknown as estypes.SearchHit<object>)
-            ) ?? [];
-
-          const processedEdges = reverse(
-            uniqBy('fields.agent_id[0]', flatten([res.edges, previousEdges]))
-          );
-
```

Comment:
Please double check, with latest changes, that this is indeed redundant.

Reply:
Great catch! I forgot about it when reimplementing public API support. Thanks @szwarckonrad , I added this back.